<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DonneesSatellite</class>
 <widget class="QMainWindow" name="DonneesSatellite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Données satellite</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_info5">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_masseSecTot">
       <property name="text">
        <string>Masse sec/totale :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="masseSecTot">
       <property name="text">
        <string notr="true">2000/3000 kg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_magnitudeStdMax">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string extracomment="Standard/maximum magnitude">Magnitude std/max :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="magnitudeStdMax">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">-0.5v/-2.9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_modele">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Modèle orbital :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="modele">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">SGP4 (NE)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_37">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>328</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_nomsat">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_nomsat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="nomsat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">ISS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_38">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_info1">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_cospar">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Désignation COSPAR :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="cospar">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">1998-067A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_39">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>271</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_info3">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_dateLancement">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Date de lancement :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_categorieOrbite">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Catégorie d'orbite :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_pays">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Pays/Organisation :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_siteLancement">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Site de lancement :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="dateLancement">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">2013/07/08</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="categorieOrbite">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">LEO/I</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="pays">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">US</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="siteLancement">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">AFETR</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_35">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>246</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_elemOsc2">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_apogee">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Apogée (Altitude) :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_perigee">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Périgée (Altitude) :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_periode">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Période orbitale :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="apogee">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">036000.0 km (036000.0 km)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="perigee">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">36000.0 km (36000.0 km)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="periode">
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string notr="true">24h 50m 53s</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_inclinaison">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Inclinaison :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="inclinaison">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">000.0000°</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>471</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_info5_2">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_classeCategDiscip">
       <property name="text">
        <string>Classe/Categorie/Discipline :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="classeCategDiscip">
       <property name="text">
        <string notr="true">C / TECH / ENV*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_dimensions">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Forme/Dimensions :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="dimensions">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">Cylindrique. L=30.0m, R=20.0m</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_40">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_info1_2">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetFixedSize</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>1</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_norad">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Numéro NORAD :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="norad">
       <property name="font">
        <font/>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string notr="true">25544</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
